modules/covid_vaccine/app/services/covid_vaccine/v0/enrollment_service.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+require 'net/sftp'
+require 'covid_vaccine/v0/expanded_registration_submission_csv_generator'
+
+module CovidVaccine
+  module V0
+    class EnrollmentService
+      def initialize
+        @records = CovidVaccine::V0::ExpandedRegistrationSubmission.enrolled_us.order(:created_at)
+        @timestamp = Time.now.utc.strftime('%Y%m%d%H%M%S')
+      end
+
+      def send_enrollment_csv
+        Net::SFTP.start(sftp_host, sftp_username, password: sftp_password) do |sftp|
+          sftp.upload!(csv_as_io, remote_file_path, name: file_name, progress: EnrollmentHandler.new)
+        end
+      end
```

Comment:
assuming this method executes without exceptions, we'll need to update the state on all of the records to pending

Reply:
this class still needs specs